src/CodeGenerator/src/Generator/ResponseParser/RestJsonParser.php

```diff
@@ -259,12 +277,12 @@ private function parseResponseMap(MapShape $shape, string $input, bool $required
         }
 
         if (!$required) {
-            $body = '!INPUT ? [] : ' . $body;
+            $body = '(!isset(INPUT) || !INPUT) ? [] : ' . $body;
```

Comment:
If input is `$array['foo']['bar']` we need to check if `"foo"` exists.

Reply:
`(!isset(INPUT) || !INPUT)` => `empty(INPUT)` ?